exercises/strings/capitalize/capitalize.js

```diff
@@ -11,14 +11,14 @@
  */
 
 function capitalize(num) {
-  // This is your job. :)
+  return string = num,
+  string.charAt(0).toUpperCase() + string.slice(1,7);
```

Comment:
Careful with the `string.slice(1, 7)`. You probably realize this, but it will only work with strings of length 7.

There are two ways to handle strings of any length:

1. Omit the second argument, which tells JS to include only characters from the index in the first argument to the end of the string. This is just how `string.slice` is built.
2. Manually specify the end with `string.length`

So the two possibilities are:

```js
string.slice(1); // everything but the first character in string
string.slice(1, string.length); // also everything but the first character in string
```